ci/travis/install-dependencies.sh

```diff
@@ -359,6 +359,12 @@ install_dependencies() {
     pip install 'recsim>=0.2.4'
   fi
 
+  # Additional Train test dependencies.
+  if [ "${TRAIN_TESTING-}" != 1 ] || [ "${DOC_TESTING-}" = 1 ]; then
```

Comment:
this should be `=1` not `!=`, right?

Reply:
Oh jeez, I don't even know how this happened.